proto/metrics/metrics.proto

```diff
@@ -710,3 +712,49 @@ message FlowlogCounters {
   // The number of failed attempts to publish a flow record.
   uint64 failed_attempts = 3;
 }
+
+// Resources used as reported by kubernetes api for the node objects.
+message KubeNodeMetrics {
+  // Name of the node
+  string name = 1;
+
+  // CPU cores in milli cores
+  uint32 cpu_cores = 2;
+
+  // CPU usage in percentage
+  float cpu_usage = 3;
+
+  // Memory usage in bytes
+  uint32 memory_usage = 4;
+
+  // Memory usage in percentage
+  float memory_usage_percentage = 5;
+}
+
+// Resources used as reported by kubernetes api for running eve user apps.
+message KubeEVEAppMetrics {
```

Comment:
Why `KubeEVEAppMetrics` and not just `KubeAppMetrics`? This is running on EVE, and an EVE app in this context is a Kubernetes app. Do we need the extra `EVE` in it?

Reply:
Yes I guess its a bit redundant, I'll shorten it

Reply:
Shorter `KubeAppMetrics` pushed up.

Reply:
Thanks.